src/ProductConstructionService/ProductConstructionService.Api/VMRConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.DarcLib.VirtualMonoRepo;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace ProductConstructionService.Api;
+
+public static class VMRConfiguration
```

Comment:
We usually name things `VmrConfiguration` with not capitalized letters

Reply:
fixed. the file name will stay capitalized tho